src/pages/index.js

```diff
@@ -1,127 +1,248 @@
+// card list elements =========================================================
 
 const addCardModal = document.querySelector(selectors.addCardModal);
-const addCardForm = addCardModal.querySelector(selectors.addCardForm);
-
-
-// Event Listeners =============================================================
+const addCardSubmitBtn = addCardModal.querySelector(selectors.modalBtn);
+const deleteCardModal = document.querySelector(selectors.deleteCardModal);
+const deleteCardModalBtn = deleteCardModal.querySelector(selectors.modalBtn);
 
+//    API    =======================================================================
 
-addCardBtn.addEventListener("click", () => {
-  addFormValidator.disableButton();
-  addCardFormPopup.open();
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/cohort-3-en",
+  headers: {
+    authorization: "97418fb8-739d-4cf9-9ff8-0802d43c1e0d",
+    "Content-Type": "application/json",
+  },
 });
 
+//     Form Validation   =====================================================================
 
-editBtn.addEventListener("click", () => {
-  const profileInfo = userInfo.getUserInfo();
-  profileTitleInput.value = profileInfo.name;
-  profileDescriptionInput.value = profileInfo.info;
+const formValidators = {};
+const enableValidation = (settings) => {
+  const formList = Array.from(document.querySelectorAll(settings.formSelector));
 
-  editFormValidator.disableButton();
+  formList.forEach((formElement) => {
+    const validator = new FormValidator(settings, formElement);
+    const formName = formElement.getAttribute("name");
 
-  profileModal.open();
-});
+    formValidators[formName] = validator;
+    validator.enableValidation();
+  });
+};
 
-//     Form =====================================================================
 
-const editFormValidator = new FormValidator(settings, profileEditForm);
-const addFormValidator = new FormValidator(settings, addCardForm);
-editFormValidator.enableValidation();
-addFormValidator.enableValidation();
+enableValidation(settings);
 
 // Functions ===================================================================
 
-  function renderCard(card) {
-    const cardElement = new Card(
-      card,
-      (imageData) => {
-        cardImagePopup.open(imageData);
-      },
-      selectors.cardTemplate
-    );
-    return cardElement.getView();
-  }
- 
-
+function setSubmitButtonText(buttonElement, text) {
+  buttonElement.textContent = text;
+}
 
+function renderCard(card, userId) {
+  const cardElement = new Card({
+    data: card,
+    handleCardClick: (imageData) => {
+      console.log(card);
+      cardImagePopup.open(imageData);
+    },
+    cardSelector: selectors.cardTemplate,
+    handleDeleteClick: () => {
+      deleteCardPopup.setAction(() => {
+        setSubmitButtonText(deleteCardModalBtn, "Deleting...");
+        api
+          .deleteCard(card._id)
+          .then(() => {
+            cardElement._handleDelete();
```

Comment:
`_handleDelete`

Please, delete the **low dash** from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.